modules/core/src/test/java/org/locationtech/jts/math/DDIOTest.java

```diff
@@ -175,7 +175,7 @@ private void checkParseError(String str) {
 		assertTrue(foundParseError);
 	}
 
-	public void tesWritetRepeatedSqrt()
+	public void testWritetRepeatedSqrt()
```

Comment:
Actually this should be `testWriteRepeatedSqrt`.  (note removal of erroneous 3rd 't').

If you change and commit that might clear up the IP validation error too.